modules/tinymce/CHANGELOG.md

```diff
@@ -6,47 +6,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Elements with only custom attributes starting with `data-` would sometimes be removed when they shouldn't #TINY-8755
-- Selecting a figure with `class="image"` would incorrectly highlight the link toolbar button #TINY-8832
+- The `@menubar-row-separator-color` oxide variable no longer affects the divider between the Menubar and Toolbar. It only controls the color of the separator lines drawn in multiline Menubars. #TINY-8632
+- The `@toolbar-separator-color` oxide variable now affects the color of the separator between the Menubar and Toolbar only. #TINY-8632
+- Available Premium plugins are listed by name in the Help dialog. These names are no longer translated. #TINY-9019
+
+### Fixed
+- The Autolink plugin did not work when text nodes in the content were fragmented. #TINY-3723
+- Fixed multiple incorrect types on public APIs found while enabling TypeScript strict mode. #TINY-8806
+- The number of blank lines returned from `editor.getContent({format: 'text'})` differed between browsers. #TINY-8579
+- The editor focused via the `auto_focus` option was not scrolled into the viewport. #TINY-8785
+- Adding spaces immediately after a `contenteditable="false"` block did not work properly in some circumstances. #TINY-8814
+- Elements with only `data-*` custom attributes were sometimes removed when they should not be removed. #TINY-8755
+- Selecting a figure with `class="image"` incorrectly highlighted the link toolbar button. #TINY-8832
 - Specifying a single, non-default list style for the `advlist_bullet_styles` and `advlist_number_styles` options was not respected. #TINY-8721
-- Fixed various issues that occurred when formatting `contenteditable` elements. #TINY-8905
-- Spaces could be incorrectly added to urlinput dialog components in certain cases #TINY-8775
-- The text pattern logic threw an error when there were fragmented text nodes in a paragraph #TINY-8779
-- Dragging a `contentEditable=false` element towards the edges would not cause scrolling #TINY-8874
-- Parsing large documents no longer throws a `Maximum call stack size exceeded` exception #TINY-6945
-- DomParser filter matching was not checked between filters, which could lead to an exception in the parser #TINY-8888
-- Lists with `contenteditable="false"` can no longer be toggled, and `contenteditable="true"` list elements within them can no longer be indented, split into another list element, or appended to the previous list element by deletion. #TINY-8920
-- Removed extra padding for the context toolbar in the `tinymce-5` skin. #TINY-8980
-- Fixed a regression where pressing Enter caused content outside the selection to be added or deleted unexpectedly. #TINY-9101
-- Fixed a bug where pressing Enter deleted selected "contenteditable="false" pre elements. #TINY-9101
+- Fixed multiple issues that occurred when formatting `contenteditable` elements with a `data-mce-cef-wrappable` attribute but no editable descendants. #TINY-8905
```

Comment:
This change is incorrect... it's not related to `data-mce-cef-wrappable` as that's new. It's related to existing formatting issues with contenteditable elements.

Reply:
Corrected an introduced error.